src/main/java/camp/nextstep/edu/step3/Hit.java

```diff
@@ -0,0 +1,46 @@
+package camp.nextstep.edu.step3;
+
+import java.util.Arrays;
+
+public enum Hit {
+    ZERO(0, 0),
+    ONE(1, 0),
+    TWO(2, 0),
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    ALL(6, 2000000000);
```

Comment:
```suggestion
    ALL(6, 2_000_000_000);
```
위와 같이 가독성을 향상시킬 수 있어요.

Reply:
오 ~ 그럼 단위를 좀 더 쉽게 이해 되겠네요 
감사합니다.
혹시 여기서 한가지 궁금한게 있는데요 한국은 3개 단위로 끊는데요 
외국 같은경우에는 그렇게 하지 않자나요 이런 경우에는 어떻게 대처해야 되는지 의견주시면 감사하겠습니다~ 👍

Reply:
외국에서도 금액은 3자리로 끊어서 쓰지 않나요 ㅎ `-` 가 세자리마다 써야 정상적으로 동작하는 것은 아니니
함께 일하는 사람들과 약속된 표현으로 사용하면 좋을것 같아요.

Reply:
네 의견 감사합니다 숫자는 끊어서 쓰도록 해야겠네요 ^^